Sources/MapboxDirections/MBDirectionsOptions.swift

```diff
@@ -219,6 +221,7 @@ public enum InstructionFormat: UInt, CustomStringConvertible {
 
  You do not create instances of this class directly. Instead, create instances of `MatchOptions` or `RouteOptions`.
  */
+@objcMembers
```

Comment:
Huh, didn’t know about this annotation. Good find. There are still some bare `@objc` keywords in the class; are they redundant now?

Reply:
`@objc` is still needed for refinements. It's also useful for compile-time errors of unbridgeable types.
Only using `@objcMembers` would fail silently if a type all of a sudden doesn't bridge to Objective-C. Haven't figured out a best practice yet, but as long as we have Obj-C bridging tests, a missing `@objc` annotation is fine since it would get caught by the test at compile-time.